CRM/Utils/Money.php

```diff
@@ -298,7 +298,7 @@ public static function formatLocaleNumericRoundedByPrecision($amount, $precision
    */
   public static function formatLocaleNumericRoundedByOptionalPrecision($amount, $precision) {
     $decimalPlaces = self::getDecimalPlacesForAmount((string) $amount);
-    $amount = self::formatUSLocaleNumericRounded($amount, $precision > $decimalPlaces ? $decimalPlaces : $precision);
+    $amount = self::formatUSLocaleNumericRounded($amount, $precision > $decimalPlaces ?: $precision);
```

Comment:
This one won't work per the test fail. Would be clearer with some brackets around it.

Reply:
Regex fail 🤕
thanks.